### DISCUSSION TITLE
Add suggestion to go to the parent definition when using `typing.override`

### DISCUSSION THREAD

**Discussion by Viicos** [microsoft/pylance-release#5282] [Enhancement]
It might be nice to have a popup allowing user to go to the parent (parents?) definition of a method when using `@override`:

```python
class A:
    def func(self):
        pass

class B:
    def func(self):
        pass

class C(B, A):
    @override  # On hover, add a goto to the parent (parents?) definition.
    def func(self):
        ...
```

This is already achievable when calling `super().func()`, but we do not necessarily call `super`.

**Comment by debonte** [comment]
I think this is already tracked somewhere. @heejaechang, do you remember where that is?

**Comment by heejaechang** [comment]
we didn't implement code lens for type/method hierarchy. implemented just `show type hierarchy` if that's what you are asking.

we can't implement `code lens` for `type/method hierarchy` due to threading/memory sharing limitation we have on js.

**Reply by heejaechang** [reply]
found issue - https://github.com/microsoft/pylance-release/issues/4972

**Comment by StellaHuang95** [comment]
If vscode has support for `go to definition` on hover, we could leverage that. But also, for go to def on `@override`, we have two potential approaches: going to definition of the decorator or the parent definition. To gather more feedback, we are moving this issue to discussion as an enhancement request for comments and upvotes.
